api.go

```diff
@@ -0,0 +1,45 @@
+package slack
+
+import (
+	"context"
+	"errors"
+	"net/url"
+)
+
+// APIResponseFull contains our API response from the api.test endpoint
+type APIResponseFull struct {
+	SlackResponse
+	Args map[string]string `json:"args,omitempty"`
+}
+
+func apiRequest(ctx context.Context, client httpClient, path string, values url.Values, debug debug) (*APIResponseFull, error) {
+	response := &APIResponseFull{}
+	err := postSlackMethod(ctx, client, path, values, response, debug)
```

Comment:
postSlackMethod's signature has changed. I suggest you rebase off the latest master.